<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RasendeRoboterClass</class>
 <widget class="QMainWindow" name="RasendeRoboterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>901</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>RasendeRoboter</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>900</height>
     </rect>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>900</width>
       <height>900</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
    <widget class="QWidget" name="lay" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>900</width>
       <height>900</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouvelle Partie</string>
    </property>
   </widget>
   <widget class="QLabel" name="goalLabel">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>60</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>47</width>
      <height>13</height>
     </size>
    </property>
    <property name="text">
     <string>But: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="solutionButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>180</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher Solution</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="nbCoups">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>140</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{background-color:black;}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>140</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombres de coups IA:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>340</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Choix du robot a d√©placer :</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-11</x>
      <y>-1</y>
      <width>1311</width>
      <height>911</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 143, 143);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="redRadioButton">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>380</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="blueRadioButton">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>440</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="greenRadioButton">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>440</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="yellowRadioButton">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>380</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton">
     <property name="geometry">
      <rect>
       <x>1079</x>
       <y>639</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="rightButton">
     <property name="geometry">
      <rect>
       <x>1160</x>
       <y>630</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="leftButton">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>630</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="upButton">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>550</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="downButton">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>710</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>110</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombres de coups joueur:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="nbCoups_2">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>110</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{background-color:black;}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="annonceNbDpl">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>210</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Faire une estimation</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="nbCoupsEstim">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>240</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{background-color:black;}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_estimation">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>240</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Estimation :</string>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>startButton</zorder>
   <zorder>goalLabel</zorder>
   <zorder>solutionButton</zorder>
   <zorder>nbCoups</zorder>
   <zorder>label</zorder>
   <zorder>widget</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>nbCoups_2</zorder>
   <zorder>annonceNbDpl</zorder>
   <zorder>nbCoupsEstim</zorder>
   <zorder>label_estimation</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RasendeRoboter.qrc"/>
 </resources>
 <connections/>
</ui>
